srcpkgs/rsvd-user/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'rsvd-user'
+pkgname=rsvd-user
+version=1.0
+revision=1
+build_style="meta"
+make_dirs="/etc/rsvd-user 755 root root"
+depends="runit"
+short_desc="User runsvdir daemons framework"
+maintainer="danoloan10 <danoloan10@tutanota.com>"
+license="Public Domain"
+homepage="danoloan.es"
```

Comment:
I couldn't access the homepage. Is that your general website or do you have a more specific page for this project? You are supposed to include `http://` or `https://` (if available) in the link as well.